listing-service/src/listing/listing.service.ts

```diff
@@ -0,0 +1,49 @@
+import { Injectable } from '@nestjs/common';
+import { PrismaService } from 'src/prisma/prisma.service';
+import { UpdateListingDto } from '../dto/update-listing.dto';
+import { GetListingDto } from '../dto/get-listing.dto';
+import { CreateListingDto } from 'src/dto/create-listing.dto';
+import { Listing, Prisma } from "@prisma/client";
+@Injectable()
```

Comment:
break a new line here to distinguish `import` statement and main class implementation.